pippo-core/src/main/java/ro/pippo/core/route/UrlResourceHandler.java

```diff
@@ -49,6 +53,63 @@ public final void handleResource(String resourcePath, RouteContext routeContext)
 
     public abstract URL getResourceUrl(String resourcePath);
 
+    protected String getResourceVersion(String resourcePath) {
+        String version = null;
+
+        URL resourceUrl = getResourceUrl(resourcePath);
+        try {
+            version =  Long.toString(resourceUrl.openConnection().getLastModified());
+        } catch (IOException e) {
+            log.error("Failed to read lastModified property for {}", resourceUrl, e);
```

Comment:
Now that I think about this some more this will probably throw an NPE when it can't locate the resource which will still trigger a `500`.  We may want to trap that.